### PR TITLE
reworked mount nfs and mount test

### PR DESCRIPTION
**Pull request by FilippChubukov**

### CODE REVIEW

File: src/fs/driver/nfs/nfs.c [hunk 1]
```diff
@@ -551,10 +551,23 @@ static int nfs_create_dir_entry(node_t *parent_node) {
...
+					predesc->file_attr.mode |= S_IFREG;
+					break;
+				case 2:
+					/* directory */
+					predesc->file_attr.mode |= S_IFDIR;
+					break;
+				case 3:
+					/* directory */
+					predesc->file_attr.mode |= S_IFDIR;
+					break;
```
Comment (alexkalmuk) [inline src/fs/driver/nfs/nfs.c]: You can write this like:

```
case 2:
case 3:
    /* directory */
    predesc->file_attr.mode |= S_IFDIR;
    break;
```

File: src/fs/driver/nfs/nfs.c [hunk 2]
```diff
@@ -551,10 +551,23 @@ static int nfs_create_dir_entry(node_t *parent_node) {
...
+				default:
+					/* unknown file type */
```
Comment (alexkalmuk) [inline src/fs/driver/nfs/nfs.c]: Can you please add `log_error("Unknown file type");` here?

File: src/tests/fs/mount_test.c [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/**
+	/**
```
Comment (alexkalmuk) [inline src/tests/fs/mount_test.c]: Extra spaces?

File: src/tests/fs/mount_test.c [hunk 4]
```diff
@@ -10,18 +10,19 @@
 #include <unistd.h>
 #include <dirent.h>
 #include <embox/test.h>
+#include <framework/mod/options.h>
 
 #include <fs/mount.h>
 
-#define FS_TEST_SHARE_PATH "10.0.2.10:/var/nfs_test"
+EMBOX_TEST_SUITE("mount nfs tests");
```
Comment (alexkalmuk) [inline src/tests/fs/mount_test.c]: You can remove `nfs` word now :)

File: src/fs/driver/nfs/nfs.c (hunk 1 above)
Comment (FilippChubukov) [reply src/fs/driver/nfs/nfs.c]: done

File: src/tests/fs/mount_test.c (hunk 3 above)
Comment (FilippChubukov) [reply src/tests/fs/mount_test.c]: removed

File: src/tests/fs/mount_test.c (hunk 4 above)
Comment (FilippChubukov) [reply src/tests/fs/mount_test.c]: done

File: src/fs/driver/nfs/nfs.c (hunk 2 above)
Comment (FilippChubukov) [reply src/fs/driver/nfs/nfs.c]: added

**Pull request merged**